client-core/src/transaction_builder.rs

```diff
@@ -38,7 +40,7 @@ pub trait WalletTransactionBuilder: Send + Sync {
         outputs: Vec<TxOut>,
         return_address: ExtendedAddr,
         attributes: TxAttributes,
-    ) -> Result<TxAux>;
+    ) -> Result<(TxAux, Vec<TxoPointer>, Coin)>;
```

Comment:
Can you update documentation with details of returned values?

Reply:
fixed